src/protagonist/Engine/Ingest/Persistence/AssetToDisk.cs

```diff
@@ -90,8 +90,8 @@ public class AssetToDisk : AssetMoverBase, IAssetToDisk
     {
         TrySetContentTypeForBinary(originResponse, asset);
         var extension = GetFileExtension(originResponse);
-        
-        var targetPath = $"{Path.Join(destinationTemplate, asset.Id.Asset)}.{extension}";
+
+        var targetPath = $"{Path.Join(destinationTemplate, asset.Id.Asset.Replace('(', '_').Replace(')', '_'))}.{extension}";

```

Comment:
Should this logic be the same as other places (where `OpenBracketReplacement` and `CloseBracketReplacement` are used, rather than hardcoded underscore).